calico/felix/test/test_config.py

```diff
@@ -237,6 +245,8 @@ def test_env_var_override(self):
         self.assertEqual(config.METADATA_PORT, 999)
         self.assertEqual(config.METADATA_IP, "1.2.3.4")
         self.assertEqual(config.STARTUP_CLEANUP_DELAY, 42)
+        self.assertEqual(config.REPORTING_INTERVAL_SECS, 10)
+        self.assertEqual(config.REPORTING_TTL_SECS, 20)
```

Comment:
These asserts are to check that the environment values were used, but you've not currently got ttl set in the environment.